schema-loader/src/main/java/schema/Table.java

```diff
@@ -0,0 +1,202 @@
+package schema;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.scalar.db.api.Scan.Ordering.Order;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.api.TableMetadata.Builder;
+import com.scalar.db.io.DataType;
+import com.scalar.db.transaction.consensuscommit.Attribute;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Table {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(Table.class);
+
+  private static final String COLUMNS = "columns";
+  private static final String TRANSACTION = "transaction";
+  private static final String PARTITION_KEY = "partition-key";
+  private static final String CLUSTERING_KEY = "clustering-key";
+  private static final String SECONDARY_INDEX = "secondary-index";
+  private static final String TRANSACTION_COL_PREFIX = "before_";
+  private static final Map<String, DataType> dataTypeMap =
+      new HashMap<String, DataType>() {
+        {
+          put("BOOLEAN", DataType.BOOLEAN);
+          put("INT", DataType.INT);
+          put("BIGINT", DataType.BIGINT);
+          put("FLOAT", DataType.FLOAT);
+          put("DOUBLE", DataType.DOUBLE);
+          put("TEXT", DataType.TEXT);
+          put("BLOB", DataType.BLOB);
+        }
+      };
+  private static final Map<String, DataType> transactionMetadataColumns =
+      new HashMap<String, DataType>() {
+        {
+          put(Attribute.COMMITTED_AT, DataType.BIGINT);
+          put(Attribute.ID, DataType.TEXT);
+          put(Attribute.PREPARED_AT, DataType.BIGINT);
+          put(Attribute.STATE, DataType.INT);
+          put(Attribute.VERSION, DataType.INT);
+        }
+      };
+  private static final Map<String, Order> orderMap =
```

Comment:
```suggestion
  private static final Map<String, Order> ORDER_MAP =
```

Reply:
Apply in [852333d](https://github.com/scalar-labs/scalardb/pull/280/commits/852333daf4d8ccda0ed3e2bed23affcd450c007a)